packages/sdk/identity/src/odis/bls-blinding-client.ts

```diff
@@ -21,23 +22,26 @@ interface BlindedMessage {
 export class WasmBlsBlindingClient implements BlsBlindingClient {
   private thresholdBls: ThresholdBlsLib
   private odisPubKey: Uint8Array
+  private dekSigner: EncryptionKeySigner
   private blindedValue: BlindedMessage | undefined
   private rawMessage: Buffer | undefined
 
-  constructor(odisPubKey: string) {
+  constructor(odisPubKey: string, dekSigner: EncryptionKeySigner) {
     this.odisPubKey = Buffer.from(odisPubKey, 'base64')
+    this.dekSigner = dekSigner
     // Dynamically load the Wasm library
     if (!this.isReactNativeEnvironment()) {
       this.thresholdBls = require('blind-threshold-bls')
     } else {
+      // TODO (Alec) Do we need to add DEK blinding to this library as well?
```

Comment:
@codyborn Do you have context on this?

Reply:
Yes, Valora uses the react-native blinding client. We should probably create a new ticket/PR to encapsulate this work.